tools/Makefile

```diff
@@ -0,0 +1,8 @@
+all: | argweaver
+	#the following tools require stand-alone compilation
+	make -C argweaver
+
+argweaver: 
+	git clone https://github.com/CshlSiepelLab/argweaver
+	# checkout a specific revision (latest as of Mar 2022)
+	cd argweaver && git checkout 7191343c1d43c02de04e4e0a150d77b0612728c6
```

Comment:
probably needs some compilation as well, or is Argweaver pure python?

Reply:
It has a Makefile inside it, and when I type `make` in the tools dir, it seems to compile ARGweaver OK. It says

```
...
Note: switching to '7191343c1d43c02de04e4e0a150d77b0612728c6'.
...
#the following tools require stand-alone compilation
make -C argweaver
g++ -c -march=core2 -mtune=haswell ...
```

Reply:
Oh, hang on, that's what the line above is doing, right?

```
all: argweaver
        #the following tools require stand-alone compilation
        make -C argweaver
```

That's how we did it in `mcveanlab/treeseq-inference`

Reply:
Ah right, that's confusing. Would be clearer to put the ``make -C argweaver`` rule at the end of the argweaver bit.